### DISCUSSION TITLE
crashreport

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#54] [Q&A]
Specific browser versions and browser built-in decoding libraries, specific video clips, such as: some Chinese comics, Ice Fire Magic Kitchen, Douluo Dalu, etc. In the middle of the playback, the speaker keeps playing, the browser dmps, and sometimes a blue screen follows. Qichu continued to follow up on the sound card driver of the little crab, and occasionally relieved it, but it doesn't work now. Hope to follow up and locate to solve this intractable disease, thank you very much!（Thorium109.0&Chrome110.0）

**Comment by gz83** [comment]
> Specific browser versions and browser built-in decoding libraries, specific video clips, such as: some Chinese comics, Ice Fire Magic Kitchen, Douluo Dalu, etc. In the middle of the playback, the speaker keeps playing, the browser dmps, and sometimes a blue screen follows. Qichu continued to follow up on the sound card driver of the little crab, and occasionally relieved it, but it doesn't work now. Hope to follow up and locate to solve this intractable disease, thank you very much!（Thorium109.0&Chrome110.0）

@whxuezhe Is there the same problem on chromium? And do you have a series of log files generated when you view the blue screen?

**Reply by whxuezhe** [reply]
@gz83 The recent crash is that when it happened; I closed the browser as soon as possible, and the blue screen did not reappear, but the browser had a 3MB dmp, which probably contained video decoding exceptions, information such as M3U8 video sources, and basic hardware information such as CPU. Previously, the blue screen was caused by the audio codec in the video, which appeared in a certain segment during playback, and was caused by the underlying conflict with the anti-virus software and sound card driver. The blue screen file can no longer be traced.

**Reply by gz83** [reply]
> @gz83 The recent crash is that when it happened; I closed the browser as soon as possible, and the blue screen did not reappear, but the browser had a 3MB dmp, which probably contained video decoding exceptions, information such as M3U8 video sources, and basic hardware information such as CPU. Previously, the blue screen was caused by the audio codec in the video, which appeared in a certain segment during playback, and was caused by the underlying conflict with the anti-virus software and sound card driver. The blue screen file can no longer be traced.

@whxuezhe Is it normal for Chromium to do the same? Also, can you upload the Log Message at the bottom of the chrome://gpu/ page?

**Reply by whxuezhe** [reply]
@gz83 
Log Messages
GpuProcessHost: The info collection GPU process exited normally. Everything is okay.
[16520:25664:1129/170450.334:WARNING:angle_platform_impl.cc(48)] : HLSLCompiler.cpp:257 (compileToBinary): C:\fakepath(159,3-70): warning X3557: loop only executes for 1 iteration(s), forcing loop to unroll
GpuProcessHost: The info collection GPU process exited normally. Everything is okay.
Log Messages
GpuProcessHost: The info collection GPU process exited normally. Everything is okay.
GpuProcessHost: The info collection GPU process exited normally. Everything is okay.
[19656:25808:1129/141646.998:ERROR:shared_image_manager.cc(189)] : SharedImageManager::ProduceSkia: Trying to Produce a Skia representation from a non-existent mailbox.
[19656:25808:1129/141647.002:ERROR:shared_image_manager.cc(189)] : SharedImageManager::ProduceSkia: Trying to Produce a Skia representation from a non-existent mailbox.

**Reply by gz83** [reply]
@whxuezhe Please re-typeset the latest reply. The current reply makes the entire webpage very long, which affects reading. At the same time, please do not upload information other than Log Message.

**Reply by whxuezhe** [reply]
@gz83 Sorry, I did what you said chrome: followed by / / gpu and then directly copy, is such a long log file.

**Reply by gz83** [reply]
> @gz83 Sorry, I did what you said chrome: followed by / / gpu and then directly copy, is such a long log file.

@whxuezhe Log Messages are located at the bottom of the chrome://gpu page, please re-upload the Log Messages recorded on the chrome://gpu page and edit them in [your reply](https://github.com/Alex313031/Thorium-AVX2/discussions/54#discussioncomment-4260270), I will check them.

**Reply by whxuezhe** [reply]
@gz83 HEVC + FFmpeg + the FFmpeg-based online streaming decoding technology used by the Opera browser before may be able to solve (fcdm1.com M3U8-HLS streaming media problem), please refer to this, thank you!

**Reply by whxuezhe** [reply]
@gz83 HEVC + FFmpeg + the FFmpeg-based online streaming decoding technology used by the Opera browser before may be able to solve (fcdm1.com M3U8-HLS streaming media problem), please refer to this, thank you!

**Reply by gz83** [reply]
> @gz83 HEVC + FFmpeg + the FFmpeg-based online streaming decoding technology used by the Opera browser before may be able to solve (fcdm1.com M3U8-HLS streaming media problem), please refer to this, thank you!

@whxuezhe The new version may adjust the functions of this part to fix problems related to media playback. I will report this situation to Alex313031. The specific effect needs to be checked in the new version.
